src/applications/vaos/appointment-list/components/tests/RequestedStatusAlert.unit.spec.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { expect } from 'chai';
+
+import { fireEvent, waitFor } from '@testing-library/react';
+
+import { Facility } from '../../../tests/mocks/unit-test-helpers';
+
+import RequestedStatusAlert from '../RequestedStatusAlert';
+import { renderWithStoreAndRouter } from '../../../tests/mocks/setup';
+
+const initialStateVAOSService = {
+  featureToggles: {
+    vaOnlineSchedulingVAOSServiceRequests: true,
+    vaOnlineSchedulingVAOSServiceCCAppointments: true,
+  },
+};
+
+const appointmentData = {
+  id: '1234',
+  location: {
+    vistaId: '983',
+    clinicId: '848',
+    stationId: '983',
+    clinicName: 'CHY PC VAR2',
+  },
+  vaos: {
+    appointmentType: 'request',
+  },
+};
+const facilityData = new Facility();
+
+describe('VAOS <RequestedAppointmentDetailsPage> with VAOS service', () => {
+  it('should display new appointment confirmation alert for VA request', async () => {
+    const appointment = {
+      ...appointmentData,
+      kind: 'clinic',
+      status: 'proposed',
+    };
+
+    const screen = renderWithStoreAndRouter(
+      <RequestedStatusAlert
+        appointment={appointment}
+        facility={facilityData}
+      />,
+      {
+        initialState: initialStateVAOSService,
+        path: `/requests/${appointment.id}?confirmMsg=true`,
+      },
+    );
+
+    expect(await screen.baseElement).to.contain('.usa-alert-success');
```

Comment:
@jenniemc - I believe we need to duplicate this it block but replace the status with canceled and test for expect(screen.baseElement).to.contain('.usa-alert-error');
that should push the coverage to 100%